aptos-move/e2e-move-tests/src/harness.rs

```diff
@@ -1331,6 +1331,7 @@ macro_rules! assert_move_abort {
                 TransactionStatus::Keep(ExecutionStatus::MoveAbort {
                     location: _,
                     code: _,
+                    message: _,
```

Comment:
does this compile? `You removed message from ExecutionStatus`, right?

Reply:
Yeah I forgot to remove this. I think this macro is not used anywhere, which is why I wasn't getting a compiler error.